src/microbe_stage/EnergyBalanceInfo.cs

```diff
@@ -18,7 +18,15 @@ public class EnergyBalanceInfo
     public float TotalProduction { get; set; }
     public float TotalConsumption { get; set; }
     public float TotalConsumptionStationary { get; set; }
+
+    /// <summary>
+    ///   The absolutely final balance of ATP when a cell is going all out and running everything and moving
```

Comment:
does this include stuff like engulfing, or is it JUST movement?

Reply:
Engulfing is not included *at all* in the energy balance info. Should I add a remark about that? Also the energy cost of binding mode is not included anywhere.

Reply:
"running everything" sure implies that it's with all possible ATP-consuming things. Other than that I don't think it's too big a deal.

Reply:
You run processes. Though I guess you don't really *run* osmoregulation
This'll get more complicated if / once we add the sprint mechanic.
I'll add a comment to try to clarify this (as I myself have found to struggle a bit to remember what all is included in all of these calculations).

Reply:
Added some clarifying comments there.